lib/components/src/blocks/Preview.tsx

```diff
@@ -54,6 +54,7 @@ const PreviewContainer = styled.div<PreviewProps>(
     position: 'relative',
     overflow: 'hidden',
     margin: '25px 0 40px',
+    display: 'inline-block',
```

Comment:
is this needed?

Reply:
It is necessary in order for the preview container to take adjust to the iFrame's width.

Without it, it would look like this:
![image](https://user-images.githubusercontent.com/48736073/70853917-f217b880-1e71-11ea-9402-3e81b47b9974.png)


Instead of this:
![image](https://user-images.githubusercontent.com/48736073/70853894-977e5c80-1e71-11ea-8b58-c701b799c2d4.png)

Reply:
Using flexbox for this is a little more robust I think.